exchanges/okgroup/okgroup_websocket.go

```diff
@@ -314,55 +357,92 @@ func (o *OKGroup) GetWsChannelWithoutOrderType(table string) string {
 // eg "spot/ticker:BTCUSD" results in "SPOT"
 func (o *OKGroup) GetAssetTypeFromTableName(table string) asset.Item {
 	assetIndex := strings.Index(table, "/")
-	return asset.Item(table[:assetIndex])
+	switch table[:assetIndex] {
+	case asset.Futures.String():
+		return asset.Futures
+	case asset.Spot.String():
+		return asset.Spot
+	case asset.PerpetualSwap.String():
```

Comment:
MY APOLOGIES! Perpetual swap won't cover things here. OKEX sends "swap" so then we get this as an ouput:
OKEX unhandled asset type swap